sample-app/authentication-sample-code/nodejs/index.js

```diff
@@ -28,6 +29,24 @@ const inputData = require('./inputData.js')
             }
         });
 
+        // If error code is 429 try to call api again upto 3 times
+        if (response.status === 429) {
```

Comment:
I was thinking of using the generateBackoffInterval functionality a little more by doing something like this:

        if (response.status === 403) {
            const interval = backOff.generateBackoffInterval(retries, backOff.getRetryInterval(response.headers.get('retry-after')));
            if (retries >= 3) {
                console.log('Cloudlink Platform is too busy!')
                process.exit();
            }
            await sleep(interval);
            console.log(`Waited for ${interval} ms`)
            console.log(`Retrying to call getCode ${retries + 1} time`)
            await getCode(userData, retries + 1)
        }

The generateBackoffInterval will actually backoff even more as the retry count gets larger.